src/OpenLoco/src/Vehicles/Vehicle.h

```diff
@@ -80,13 +80,15 @@ namespace OpenLoco::Vehicles
 
     struct Vehicle2or6;
 
-    namespace Flags5F
+    enum class Flags5F : uint8_t
```

Comment:
Lets call these `BreakdownFlags` for now.  And change var_5F to `breakdownFlags`